db/migrate/20220120164746_add_accion_to_curiosity.rb

```diff
@@ -0,0 +1,5 @@
+class AddAccionToCuriosity < ActiveRecord::Migration[7.0]
+  def change
+    add_column :curiosity_cards, :action, :integer
```

Comment:
This could be done on `CreateCuriosityCards` migration

Reply:
same here, but in this part, I created an enum with a migration :D